app/serializers/email_serializer.rb

```diff
@@ -1,35 +1,10 @@
 # frozen_string_literal: true
 
 class EmailSerializer < ActiveModel::Serializer
-  attribute :email
-  attribute :effective_at
+  attribute :email_address
+  attribute :effective_date
```

Comment:
This attribute remapping/naming was collaborated on with the dragons FE team.  Before committing these changes, you'll need to confirm with them.  It will require that they refactor things on their end to prevent breakage.

Heads up, this feature is about to go into UAT.  Might be a challenging time for the API to change.

If the attrs do end up changing, you'll also need to update the swagger docs.

Reply:
I see.  I'll change them back.  I don't want to create issues for the other team for a trivial change like this.